test/package/helpers.test.js

```
@@ -9,3 +9,66 @@
+  
+  beforeEach(() => {
+    // Reset the mock before each test
+    jest.resetModules()
+  })
+
+  test("should not throw when Babel core packages are present", () => {
+    // Mock that both core packages exist
+    const helpers = require("../../package/utils/helpers")
+    helpers.moduleExists = jest.fn((pkg) => {
+      return pkg === "@babel/core" || pkg === "babel-loader"
+    })
```

ESLint

[error] Use jest.spyOn() instead

(jest/prefer-spy-on)